src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -1598,16 +1653,17 @@ public enum HolonPhantoms implements LogicCardInfo {
           onAttack {
             damage 30
 
-            if (self.cards.findAll { it.name.contains("Holon Energy") }) {
+            if (self.cards.any{ it.name == "Holon Energy" }) {
               discardRandomCardFromOpponentsHand()
             }
           }
         }
         move "Swift Turn", {
-          text "50+ damage. If the Defending Pokémon has Fighting Resistance, this attack does 50 damage plus 30 more damage."
+          text "50+ damage. If the Defending Pokémon has [F] Resistance, this attack does 50 damage plus 30 more damage."
```

Comment:
These also need corrected in carddb in most cases.

Reply:
Yeah, haven't been doing that. Will start updating it as I find any 👍

Reply:
https://github.com/tcgone/carddb/pull/62